packages/docs/developer-resources/start.md

```diff
@@ -1,11 +1,30 @@
 # Start
 
-The following examples will serve as a guide to understand the Celo SDK:
+The following examples and guides will help you start building on Celo.
 
-* The [Sending CELO](walkthroughs/hellocelo.md) walkthrough will guide you through the basics of connecting to Celo networks using ContractKit.
+## Backend / command line interface
```

Comment:
I think a better heading would be `Interacting with Smart Contracts and Celo Networks`
Backend is a bit of a vague term and means something slightly different to everyone (e.g. someone who isn't crypto-native may not know that 'backend' includes smart contract interaction)